### PR TITLE
Documentation: Transit Skimming and Assignment

### PR DESCRIPTION
**Pull request by JiaXu1024**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Impact

Improved documentation

## Types of changes

What types of changes does your code introduce to ABM?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation Update (if none of the other choices apply)

## How has this been tested?
Please describe the tests that you ran to verify your changes.

- [X] Built the documentation locally and confirmed the updates

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

## Further comments

None

### CODE REVIEW

File: docs/design/supply/transit-skims-assign.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
+
+The shortest "travel time" is a generalized cost formulation, including perception factors (or weights) on the different travel time components, along with fares, and other costs / perception biases such as transfer penalties which vary over the network and transit journey.
+
+The model has three access modes to transit (walk, park-and-ride (PNR), and kiss-and-ride (KNR)) and three transit sets (local bus only, premium transit only, and local bus and premium transit sets), for 9 total demand classes by 5 TOD. These classes are assigned by slices, one at a time, to produce the total transit passenger flows on the network.
```
Comment (bhargavasana) [inline docs/design/supply/transit-skims-assign.md:9]: We now have TNC-to-transit and also microtransit/NEV-to-transit.

Comment (JiaXu1024) [reply docs/design/supply/transit-skims-assign.md:9]: > We now have TNC-to-transit and also microtransit/NEV-to-transit.

In the output of Residential Transit Trip Matrix, we have WALK, TNC, KNR and PNR set, but no microtransit/NEV set, shall we say four access modes?

**Review by JoeJimFlood — APPROVED** [review]

File: docs/design/supply/transit-skims-assign.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+While there are 9 slices of demand, there are only three classes of skims: Local bus only, premium only, and all modes. The access mode does not change the assignment parameters or skims.
```
[the remark this answers is not in the document]
Comment (JiaXu1024) [reply docs/design/supply/transit-skims-assign.md]: > I am thinking drive-transit skims are separate than walk-transit skims. Recently we have been talking about recording the PNR/KNR station if that is possible in EMME.

Currently we have transit skims by 5 time periods that include three transit classes: LOC, PRM and LOC. We can update this part once we have skims updates done later.

**Review by anneku — APPROVED** [review]

File: docs/design/supply/transit-skims-assign.md (hunk 1 above)
Comment (bhargavasana) [reply docs/design/supply/transit-skims-assign.md]: I will merge this in for now but we need to push another update to this section soon.

File: docs/design/supply/transit-skims-assign.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bhargavasana) [reply docs/design/supply/transit-skims-assign.md]: I will merge this in for now but we need to push another update to this section soon.

**Pull request merged**